esmvalcore/local.py

```diff
@@ -302,11 +293,15 @@ def select_files(filenames, timerange):
     Otherwise, the file selection occurs taking into account
     the time resolution of the file.
     """
+    if '*' in timerange:
+        # TODO: support * combined with a period
+        return filenames
+
```

Comment:
In what instance do you run into this case? For what I have seen the wildcard in the time range gets substituted before reaching this point.

Reply:
When someone would use the function `esmvalcore.local.find_files` directly instead of from the recipe

Reply:
I'll add a note to the docstring then

Reply:
Done in [b9e224a](https://github.com/ESMValGroup/ESMValCore/pull/1835/commits/b9e224a820282d82b8982281273414cc7c75d2c0)